File: spec/code_samples/Php_http2/profiles@documents/delete.php
<?php

$client = new http\Client;
$request = new http\Client\Request;

$request->setRequestUrl('https://www.directfreight.com/api/profiles/documents');
$request->setRequestMethod('DELETE');
$request->setHeaders(array(
  'end-user-token' => 'SOME_STRING_VALUE'
));

$client->enqueue($request)->send();
$response = $client->getResponse();

echo $response->getBody();

File: spec/code_samples/Python_python3/postings@custom_categories@{category_type}/delete.py
import http.client

conn = http.client.HTTPSConnection("www.directfreight.com")

headers = { 'end-user-token': "SOME_STRING_VALUE" }

conn.request("DELETE", "/api/postings/custom_categories/%7Bcategory_type%7D?category_id=SOME_ARRAY_VALUE", headers=headers)

res = conn.getresponse()
data = res.read()

print(data.decode("utf-8"))